pkg/kubelet/volume/gce_pd/mount_util.go

```diff
@@ -35,19 +39,36 @@ func getMountRefCount(mounter mount.Interface, mountPath string) (string, int, e
 
 	// Find the device name.
 	deviceName := ""
-	for i := range mps {
-		if mps[i].Path == mountPath {
-			deviceName = mps[i].Device
+	for _, mp := range mps {
```

Comment:
And here I thought I was cleaning stuff up...
Good thing to keep in the back of my head. Changed all of these back.